study.md

```diff
@@ -46,14 +46,24 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+I don't think it's anything special... just adds an exclamation mark after the string.
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+class = Greeter
+  def initialize=(name="World")
+    @name = name
+  end
+  def say_hi
+    puts "Hi #{@name}!"
+  end
+  def say_bye
+    puts "Cya #{@name}!"
+  end
+end
```

Comment:
This is defining a class not creating an instance of a class:

`my_instance = MyClass.new`